From: Jens Axboe <axboe@kernel.dk>
Date: Fri, 3 Nov 2017 11:00:03 -0600
Subject: [PATCH] cdrom: hide CONFIG_CDROM menu selection
Git-commit: a116895fc7b61de4f26548c9b635a06515ec7563
Patch-mainline: v4.15-rc1
References: bsc#1104967,FATE#325924

We don't need to expose this. The point is that drivers select
the uniform CDROM layer, if they need it, the user should not
have to make a conscious decision on whether to include this
separately or not.

Fixes: 2a750166a5be ("block: Rework drivers/cdrom/Makefile")
Signed-off-by: Jens Axboe <axboe@kernel.dk>
Acked-by: Hannes Reinecke <hare@suse.com>
---
 drivers/block/Kconfig | 9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

diff --git a/drivers/block/Kconfig b/drivers/block/Kconfig
index b998b3c9ea86..95e678716b5e 100644
--- a/drivers/block/Kconfig
+++ b/drivers/block/Kconfig
@@ -68,14 +68,7 @@ config AMIGA_Z2RAM
 	  module will be called z2ram.
 
 config CDROM
-	tristate "CD-ROM driver"
-	help
-	  A CD-ROM is a pre-pressed optical compact disc which contains
-	  data. The name is an acronym which stands for "Compact Disc
-	  Read-Only Memory". This driver implements functionality that is
-	  shared by all CD-ROM drivers, e.g. the IDE CD-ROM driver and the
-	  SCSI CD-ROM driver. For a list of the ioctls implemented by this
-	  driver, see also Documentation/ioctl/cdrom.txt.
+	tristate
 
 config GDROM
 	tristate "SEGA Dreamcast GD-ROM drive"
-- 
2.16.4

